content/guides/local-hardening/_index.md

```diff
@@ -22,10 +22,28 @@ Note: On some systems, the python binary is called `python` and not `python3`.
 We assume your project was cloned to `~/factory-test`. If you don't have a project yet, and want to follow these instructions to prepare your local setup, you can clone the `factory-test` repo by running the following command:
 
 ```sh
-git clone https://github.com/TinyTapeout/tt08-factory-test ~/factory-test
+git clone https://github.com/TinyTapeout/tt09-factory-test ~/factory-test
 ```
 
-#### 1. Set up environment variables
+### 2. Python and Pip Dependencies
+
+Create a dedicated directory for the virtual Python environment and initialize it:
+
+```sh
+mkdir ~/ttsetup
+python3 -m venv ~/ttsetup/venv
+source ~/ttsetup/venv/bin/activate
+```
+
+Then install the dependencies:
+
+```sh
+pip install -r ~/factory-test/tt/requirements.txt
```

Comment:
This refers to `~/factory-test/tt` before it's created by git clone way later in the document.

Reply:
thanks - I fixed it.